moveit_core/robot_state/test/test_kinematic.cpp

```diff
@@ -553,6 +558,99 @@ TEST(FK, OneRobot)
   EXPECT_NEAR_TRACED(state.getGlobalLinkTransform("link_e").translation(), Eigen::Vector3d(2.8, 0.6, 0));
 }
 
+class LoadPR2 : public testing::Test
+{
+protected:
+  virtual void SetUp()
```

Comment:
function names start lower case

Reply:
This is a gtest thing https://github.com/google/googletest/blob/master/googletest/docs/Primer.md#test-fixtures-using-the-same-data-configuration-for-multiple-tests

Reply:
thanks for the link